sklearn/plot/_gridsearch_results.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
+from sklearn.plot import plot_heatmap
+
+
+def plot_gridsearch_results(cv_results, metric='mean_test_score',
```

Comment:
or we can probably modify [this example](http://scikit-learn.org/dev/auto_examples/model_selection/plot_grid_search_digits.html#sphx-glr-auto-examples-model-selection-plot-grid-search-digits-py), add the plot here instead of the table.
can we simply do it on roc-aoc instead of precision recall? Then we can show 2 cases
1. just using a single parameter
2. using 2 parameters